CONTRIBUTORS

```diff
@@ -5,3 +5,4 @@ uStudio, Inc
     Ricardo Contreras
     Thomas Stephens
     Josh Marshall
+    Brice Grichy
```

Comment:
@kavod Could you add a blank line above your name, and not indent it, to keep it separated from the `uStudio, Inc` "section"? Thanks.